tests/cleanup_dummy_device.yml

```diff
@@ -0,0 +1,13 @@
+---
+- name: Clean up dummy testing device
+  file:
+    path: "{{ test_device }}"
```

Comment:
please prefix variables used in tests with `nbde_client_`

Reply:
One reason is that we may write integration tests with several roles in a single playbook

Reply:
I will update this as well.